intranet/apps/bus/management/commands/import_routes.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=consider-using-with
```

Comment:
Instead of disabling `consider-using-with` could we use

```python
with open(...) as ...:
```

?

Reply:
So `with csv.reader(open("routes.csv", "r")) as reader:` doesn't address this concern; it is worried about the `open("routes.csv", "r")` in the middle. We'd have to do something like:
```
with open("routes.csv", "r") as f:
    reader = csv.reader(f)
```
I didn't think that was very clean, but we could do that.